### PR TITLE
LUN-98: Adjust folder structure

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: docs/technical-documentation.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- [Folder structure](#Folder structure)
+
+## Folder structure
+
+```
+src
+└───routes
+│ └───someRoute
+│ │   │ SomeRoutePage.tsx
+│ └───component
```
Comment (klinzo) [inline docs/technical-documentation.md:14]:
```suggestion
│ └───components
```

File: docs/technical-documentation.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+│ │   │ SomeRouteItem.tsx
+│ │   │ SomeRouteList.tsx
+│ └───service
```
Comment (klinzo) [inline docs/technical-documentation.md:17]:
```suggestion
│ └───services
```

File: src/components/Modal.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import Button from './Button'
 import { BUTTONS_THEME } from '../constants/data'
 import { COLORS } from '../constants/colors'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
-import AsyncStorage from '../utils/AsyncStorage'
+import AsyncStorage from '../services/AsyncStorage'
```
Comment (klinzo) [inline src/components/Modal.tsx:8]: In integreat we'll do the other way, currently its services but we are planning on changing it to utils as almost none are real services.

**Review by klinzo — APPROVED** [review]

File: src/components/Modal.tsx (hunk 3 above)
Comment (SteffiStuffel) [reply src/components/Modal.tsx:8]: hmh, yeah thought about this also. So maybe i keep it as it was. Not sure though.

File: docs/technical-documentation.md (hunk 1 above)
Comment (SteffiStuffel) [reply docs/technical-documentation.md]: done

File: docs/technical-documentation.md (hunk 2 above)
Comment (SteffiStuffel) [reply docs/technical-documentation.md]: done

**Pull request merged**